mmf/utils/logger.py

```diff
@@ -1,174 +1,251 @@
 # Copyright (c) Facebook, Inc. and its affiliates.
 
 import collections
```

Comment:
Will be good to add a comment here that most of this is adapted from Detectron2.